executor/builder.go

```diff
@@ -3313,8 +3313,12 @@ func (b *executorBuilder) buildWindow(v *plannercore.PhysicalWindow) *WindowExec
 	windowFuncs := make([]aggfuncs.AggFunc, 0, len(v.WindowFuncDescs))
 	partialResults := make([]aggfuncs.PartialResult, 0, len(v.WindowFuncDescs))
 	resultColIdx := v.Schema().Len() - len(v.WindowFuncDescs)
+	frameUnBounded := false
```

Comment:
I think we should not do extra frame operation here.
This logic can be handled in [handleDefaultFrame](https://github.com/pingcap/tidb/blob/889eb98e8e4c223d0550921bcee5c19722dc51a0/planner/core/logical_plan_builder.go#L4266). Then each partition will use `aggWindowProcessor` but not `rowFrameWindowProcessor` to calculate and the sliding window will not be used for better performance.

Reply:
i will fix it